lib/database/calendar.ts

```diff
@@ -1,5 +1,7 @@
 import { client } from "../db";
```

Comment:
Forgot about this. can we put this in calendar-events.ts? That way the file the database functions are in correspond to the route they belong to.